sdk/communication/Azure.Communication.Chat/tests/SessionRecords/ChatClientsLiveTests/ReadReceiptGS.json

```diff
@@ -1,19 +1,16 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://azurecommunicationservices.communication.azure.com/identities?api-version=2021-03-07",
+      "RequestUri": "https://chat-sdktester-e2e.communication.azure.com/identities?api-version=2021-03-07",
```

Comment:
It seems that `ChatRecordedTestSanitizer` does not inherit from `CommunicationRecordedTestSanitizer`? Let's do that to get a bit more sanitization

Reply:
We need a correctly formatted token as part of the calls to the API along with the ids for the tests, and likely re-write some of the current logic (ids , urls and IP address are not PII or PUI, so we are good in terms of security). We could try some overwrittings on the ChatRecordedTestSanitizer itself but will look at it when we have some extra time (after GA).